packages/gatsby/src/utils/__tests__/test-require-error.js

```diff
@@ -37,4 +37,36 @@ describe(`test-require-error`, () => {
       )
     }
   })
+
+  describe(`handles error message thrown by Bazel`, () => {
+    it(`detects require errors`, () => {
+      const bazelModuleNotFoundError = new Error(`Error: //:build_bin cannot find module './fixtures/module-does-not-exist' required by '/private/var/tmp/_bazel_misiek/eba1803983a26276494495d851e478a5/execroot/com_github_bweston92_debug_gatsby_bazel_rules_nodejs/bazel-out/darwin-fastbuild/bin/build.runfiles/com_github_bweston92_debug_gatsby_bazel_rules_nodejs/node_modules/gatsby/dist/bootstrap/get-config-file.js'
+      looked in:
+        built-in, relative, absolute, nested node_modules - Error: Cannot find module './fixtures/module-does-not-exist'
+        runfiles - Error: Cannot find module '/private/var/tmp/_bazel_misiek/eba1803983a26276494495d851e478a5/execroot/com_github_bweston92_debug_gatsby_bazel_rules_nodejs/bazel-out/darwin-fastbuild/bin/build.runfiles/fixtures/module-does-not-exist'
+        node_modules attribute (com_github_bweston92_debug_gatsby_bazel_rules_nodejs/node_modules) - Error: Cannot find module '/private/var/tmp/_bazel_misiek/eba1803983a26276494495d851e478a5/execroot/com_github_bweston92_debug_gatsby_bazel_rules_nodejs/bazel-out/darwin-fastbuild/bin/build.runfiles/com_github_bweston92_debug_gatsby_bazel_rules_nodejs/node_modules/fixtures/module-does-not-exist'`)
```

Comment:
I feel silly about inlining this, but spinning env with Bazel would be much more costly than hard coding those errors

Reply:
I’m just happy it will allow others to use Gatsby with Bazel :) thanks @pieh !